blog/2018/12/21/a-little-christmas-template-cooking/index.markdown

```diff
@@ -0,0 +1,104 @@
+---
+status: published
```

Comment:
You probably want `disable_content_template: 1` as seen https://github.com/MojoliciousDotIO/mojolicious.io/blob/master/blog/2018/12/02/automatic-reload-for-rapid-development/index.markdown for example, so that statocles doesn't try to use the template mark in your examples.